modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -3,87 +3,91 @@
-
-. Configure SSH on the Windows VM by running the following Powershell script:
+. Create a new virtual machine in the vSphere client from the Windows Server 2004 ISO image. See link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.hostclient.doc/GUID-7834894B-DD17-4D59-A9BF-A33D02478521.html[VMware's documentation] for more information.
 +
-[source,posh]
-----
-# Powershell script to configure SSH on vSphere Windows VMs
-Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
-$firewallRuleName = "ContainerLogsPort"
-$containerLogsPort = "10250"
-New-NetFirewallRule -DisplayName $firewallRuleName -Direction Inbound -Action Allow -Protocol TCP -LocalPort $containerLogsPort -EdgeTraversalPolicy Allow
-Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
-Install-Module -Force OpenSSHUtils
-Set-Service -Name ssh-agent -StartupType 'Automatic'
-Set-Service -Name sshd -StartupType 'Automatic'
-Start-Service ssh-agent
-Start-Service sshd
-$pubKeyConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PubkeyAuthentication yes','PubkeyAuthentication yes'
-$pubKeyConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$passwordConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PasswordAuthentication yes','PasswordAuthentication yes'
-$passwordConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$authFileConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace 'AuthorizedKeysFile __PROGRAMDATA__/ssh/administrators_authorized_keys','#AuthorizedKeysFile __PROGRAMDATA__/ssh/administrators_authorized_keys'
-$authFileConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$pubKeyLocationConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace 'Match Group administrators','#Match Group administrators'
-$pubKeyLocationConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-Restart-Service sshd
-New-item -Path $env:USERPROFILE -Name .ssh -ItemType Directory -force
-----
+[IMPORTANT]
+====
+The Windows Server 2004 VM must include the link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch KB4565351]. This patch is required to set the VXLAN UDP port, which is required for clusters installed on vSphere. This patch is not available for the `Windows Server 1809` VM image.
+====
 
 . Install and configure VMware Tools version 11.0.6 or greater on the Windows VM. See the link:https://docs.vmware.com/en/VMware-Tools/index.html[VMware Tools documentation] for more information.
```

Comment:
Please an installation specify [target link ](https://docs.vmware.com/en/VMware-Tools/11.2.0/com.vmware.vsphere.vmwaretools.doc/GUID-D8892B15-73A5-4FCE-AB7D-56C2C90BD951.html)

Reply:
@jrvaldes shouldn't we be point to the [main doc](https://docs.vmware.com/en/VMware-Tools/index.html) rather than the installation for [version 11.2.0](https://docs.vmware.com/en/VMware-Tools/11.2.0/com.vmware.vsphere.vmwaretools.doc/GUID-D8892B15-73A5-4FCE-AB7D-56C2C90BD951.html)?

Reply:
@jrvaldes I was aiming to keep this somewhat generic to avoid committing to a specific version of VMware Tools installation. If we want to recommend a specific version to install, though, I can point to a version-specific install link. Let me know.

Reply:
Good point, disregard my comment. Keep it generic.